### DISCUSSION TITLE
Document store upsert

### DISCUSSION THREAD

**Discussion by torontopho** [FlowiseAI/Flowise#3282] [General]
Am trying to upsert my vector from the document store , and its giving an erro 

- I have a paid plan fr the cloud version of flowise
- i have dropped same issue on discord , no respond 

Pls can i get this solved here 

![flowise issue](https://github.com/user-attachments/assets/6cd27145-f735-4c63-82b7-ce73bd01aa2e)

**Comment by torontopho** [comment]
Figures that out , it was my open ai token i used fr my embeddings that ws incorrect

**Reply by HenryHengZJ** [reply]
Glad you figured out! You can also reach us at support@flowiseai.com
